create-neon/src/shell.ts

```diff
@@ -15,19 +15,38 @@ export default function shell(cmd: string, args: string[], cwd: string): Promise
     reject = rej;
   });
 
-  child.on('exit', (code) => {
+  function deleteNeonDir(): Promise<void> {
+    return fs.rm(cwd, { force: true, recursive: true });
+  }
+
+  child.on("exit", async (code) => {
     if (code == null) {
-      process.exit(1);
+      await deleteNeonDir();
+      process.exit();
     }
     if (code !== 0) {
+      await deleteNeonDir();
       process.exit(code);
     }
+
+    if (code === 0) {
+      try {
+          let data = await fs.readFile(`${cwd}/package.json`,'utf8')
+          let { description, author, license } = JSON.parse(data);
+          if ([description, author, license].includes(undefined)) {
```

Comment:
Are these fields still defined if you hit `enter` without entering anything?

Reply:
Yes, I tried doing the same with npm versions latest, LTS and a really old one 4.9.1 . Functionality is same and these fields do exist if we press `enter` without entering anything.